### PR TITLE
created: favourite movie data base

### PR DESCRIPTION
**Pull request by mmusk**

### CODE REVIEW

File: assets/scripts/app.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const deleteMovieModal = document.getElementById('delete-modal');
+
+const movies = [];
+
+const toggleBackdrop = () => {
+  backdrop.classList.toggle('visible');
+};
+
+const updateUI = () => {
+  if (movies.length === 0) {
```
Comment (Tanvir04khan) [inline assets/scripts/app.js]:
```suggestion
  if (!movies.length) {
```

File: assets/scripts/app.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+const addMovieHandler = () => {
+  const titleValue = userInputs[0].value;
+  const imageUrlValue = userInputs[1].value;
+  const ratingValue = userInputs[2].value;
+
+  if (
+    titleValue.trim() === '' ||
+    imageUrlValue.trim() === '' ||
+    ratingValue.trim() === '' ||
```
Comment (Tanvir04khan) [inline assets/scripts/app.js]: Checking for falsey value, can you use shorthand? For line no. 111, 112 and 113

File: assets/scripts/app.js (hunk 1 above)
Comment (mmusk) [reply assets/scripts/app.js]: fixed @Tanvir04khan

File: assets/scripts/app.js (hunk 2 above)
Comment (mmusk) [reply assets/scripts/app.js]: fixed @Tanvir04khan

**Review by syedabdullahali — APPROVED** [review]

**Review by anask601 — APPROVED** [review]
lgtm

**Review by Tanvir04khan — APPROVED** [review]

**Pull request merged**